src/api/yorkie/v1/admin.proto

```
@@ -17,7 +17,7 @@ syntax = "proto3";
 
 package yorkie.v1;
 
-import "src/api/yorkie/v1/resources.proto";
+import "yorkie/v1/resources.proto";
```

Buf (1.66.0)

[error] import "yorkie/v1/resources.proto": file does not exist

(COMPILE)